src/langsmith/integrations.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Integrations
+sidebarTitle: Overview
+mode: wide
+---
+
+LangSmith provides support for [LangChain](/oss/python/langchain/overview) and [LangGraph](/oss/python/langgraph/overview) as well as integrations with a growing set of popular [LLM providers](#llm-providers) and [agent frameworks](#agent-frameworks). For setup and usage, refer to the guide pages in the navigation bar.
+
+## Open source frameworks
```

Comment:
@lnhsingh I removed "native" not a huge fan of that. I don't know if we want to somehow define that these are kind of part of the product?

Also, I didn't include the hyphen, only because I think even before the noun "open source" is kept open as it is widely known. I super don't mind know if you have strong opinions on that!!!